cloud_config/README.md

```diff
@@ -0,0 +1,21 @@
+This directory contains the extra files required to deploy Matatika-CE into cloud or as a server (application is accessible from outside the server deployment. Not in localhost)
+
+`app_certificate/privkey.pem`  : the private key for your certificate. - for the application
+`app_certificate/fullchain.pem`: the certificate file used in most server software.  - for the application
+`catalog_certificate/privkey.pem`  : the private key for your certificate. - for the catalog
+`catalog_certificate/fullchain.pem`: the certificate file used in most server software.  - for the catalog
+
+Presequisite: 
+- NGINX is installed in the VM.
+
+Steps to run the cloud version:
+
+1. Generate the SSL certificates for app and catalog:
```

Comment:
We will get questions about how.  Adding links to letsencrypt instructions, or copy an paste our normal instructions could be good.